client/src/components/users/EditUserProfileWrapper.tsx

```diff
@@ -0,0 +1,106 @@
+import React, {Component} from "react";
+import {match} from "react-router";
+import {Query} from "react-apollo";
+import gql from "graphql-tag";
+import {Header, Loader, Message} from "semantic-ui-react";
+import {User} from "../../actions";
+import UserProfile from "./UserProfile";
+import {AppState} from "../../reducers/reducers";
+import {connect} from "react-redux";
+
+interface EditUserProps {
+    match: match & EditUserParams;
+    user: User | null;
+}
+
+interface EditUserParams {
+    params: { userId?: string };
+}
+
+
+const USER_QUERY = gql`
+    query users($uuid:String!) {
+        users(search:{uuid:$uuid}) {
+            uuid
+            name
+            email
+            phone
+            slackUsername
```

Comment:
will bolt accounts require slack handles?

Reply:
Yes, I think it's useful